Basic-Video-Zoom/README.md

```diff
@@ -0,0 +1,54 @@
+OpenTok.js Basic Video Zoom
+=======================
+
+This sample application shows how to use a basic transformer to zoom the publisher with the Vonage
+video APIs.
+
+## Demo
+
+You can see a demo of this sample running at [opentok.github.io/opentok-web-samples/Basic%20Video%20/Zoom/](https://opentok.github.io/opentok-web-samples/Basic%20Video%20Zoom/)
```

Comment:
This needs to be changed to the following:
* opentok.github.io/opentok-web-samples/Basic%20Video%20/Zoom/ -> opentok.github.io/opentok-web-samples/Basic-Video-Zoom/
* https://opentok.github.io/opentok-web-samples/Basic%20Video%20Zoom/ -> https://opentok.github.io/opentok-web-samples/Basic-Video-Zoom/

as the directory is named Basic-Video-Zoom